lib/msal-browser/src/crypto/CryptoOps.ts

```diff
@@ -95,9 +94,24 @@ export class CryptoOps implements ICrypto {
      * Generates a keypair, stores it and returns a thumbprint
      * @param request
      */
-    async getPublicKeyThumbprint(request: SignedHttpRequestParameters): Promise<string> {
+    async getPublicKeyThumbprint(request: SignedHttpRequestParameters, keyType?: CryptoKeyTypes): Promise<string> {
+        let keyOptions: CryptoKeyOptions;
+
+        switch(keyType) {
+            case CryptoKeyTypes.StkJwk:
+                keyOptions = CryptoKeyConfig.RefreshTokenBinding;
+                break;
+            default:
+                keyOptions = CryptoKeyConfig.AccessTokenBinding;
+                break;
+        }
```

Comment:
I think a ternary would be cleaner here (unless if you think there will be more types here in the future).

Reply:
I did originally add the switch to allow for other key types in the future, but we can just extend the ternary to a switch when that happens, so keeping it cleaner may be wiser.